src/Ball.java

```diff
@@ -93,4 +95,31 @@ public void move() {
         }
     }
 
+    /** This method switched the color, x and y speed of both ball after colliding */
```

Comment:
Any source for this? We may have discussed it in class, I honestly don't remember, but whoever you learned this trick from or wherever you got the idea should be cited or at least mentioned.